base64ct/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "b64ct"
+name = "base64ct"
 version = "0.2.0" # Also update html_root_url in lib.rs when bumping this
```

Comment:
Is it worth to do the initial release under v0.2.0 and not v0.1.0?

Reply:
Oops, I didn't change the version number.

Yeah, I can bump it back down before an initial release.